File: frontend/types/dashboard/ReviewPostDto.ts
export interface ReviewPostDto {
  id: number;
  no: string;               // 백엔드에서 내려오는 순위 ("01", "02", "03")
  title: string;
  author: string;
  content: string;
  regionName: string;       // 백엔드에서 regionId 기반으로 생성된 지역명
  regionId: number;
  thumbnailUrl?: string | null;
  likes: number;
  views: number;
  createdAt: string;        // ISO 문자열 (서버에서 LocalDateTime→ISO)
  summary: string;
}

// 프론트에서 사용하는 타입 (백엔드와 동일하므로 별칭으로 사용)
export interface ReviewPostWithRank extends ReviewPostDto {}

File: backend/src/main/java/com/jibangyoung/domain/dashboard/dto/ReviewPostDto.java
package com.jibangyoung.domain.dashboard.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ReviewPostDto {
    private Long id;
    private String no;
    private String title;
    private String author;
    private String content;
    private String regionName; // 프론트엔드와 호환을 위해 regionName 사용
    private Long regionId;
    private String thumbnailUrl;
    private Integer likes;
    private Integer views;
    private String createdAt;
    private String summary;
}